src/MIT.xml

```diff
@@ -9,7 +9,7 @@
          <p>MIT License</p>
       </titleText>
       <copyrightText>
-         <p>Copyright (c) &lt;year&gt; &lt;copyright holders&gt;</p>
+         <p><alt match=".+" name="copyright">Copyright (c) &lt;year&gt; &lt;copyright holders&gt;</alt></p>
```

Comment:
This seems overly broad.  Can we follow [the BSD-2-Clause example][1] and keep "Copyright (c) " required?

[1]: https://github.com/spdx/license-list-XML/blob/v3.1/src/BSD-2-Clause.xml#L10-L11